pkg/forwarder/domain_forwarder.go

```diff
@@ -135,12 +147,39 @@ func (f *domainForwarder) retryTransactions(retryBefore time.Time) {
 	}
 }
 
+func (f *domainForwarder) tryAddToTransactionContainer(t Transaction) (bool, int) {
+	if f.optionalTransactionContainer == nil {
+		return false, 0
+	}
+
+	dropCount, err := f.optionalTransactionContainer.Add(t)
+	if err != nil {
+		log.Errorf("Error when adding a transaction to the retry queue: %v", err)
+	}
+
+	if dropCount > 0 {
+		transactionEndpointName := t.GetEndpointName()
+		transactionsDroppedByEndpoint.Add(transactionEndpointName, int64(dropCount))
+		transactionsDropped.Add(int64(dropCount))
+		tlmTxDropped.Inc(f.domain, transactionEndpointName)
```

Comment:
Shouldn't it be increased by dropCount instead of 1?

Reply:
Code updated.